superset-frontend/cypress-base/cypress/integration/dashboard/edit_mode.test.js

```diff
@@ -23,34 +23,35 @@ describe('Dashboard edit mode', () => {
     cy.server();
     cy.login();
     cy.visit(WORLD_HEALTH_DASHBOARD);
+    cy.route('POST', '/superset/explore_json/*?dashboard_id=1').as('worldMap');
     cy.get('.dashboard-header [data-test=pencil]').click();
   });
 
   it('remove, and add chart flow', () => {
-    // wait for box plot to appear
-    cy.get('.grid-container .box_plot');
-
-    cy.get('.fa.fa-trash')
-      .last()
+    // wait for world map to appear
+    cy.wait('@worldMap');
+    cy.get('.world_map', { timeout: 60000 })
```

Comment:
Why wait 1 minute? If I'm understanding Cypress right, after you waited for the API requests, the additional wait time is only about UI render time. No matter how long you wait, the element may still be undiscoverable to Cypress if something is broken.

Reply:
I do not think the flaky problem is caused by `box_plot` chart

Reply:
@graceguo-supercat I agree, I don't think the `box_plot` is the issue. Apologies for the confusion, I posted this to test the ci while working out CSRF on Cypress and docker together locally. Will update this test here in a bit.